pkg/v1/tkg/client/upgrade_cluster.go

```diff
@@ -86,6 +89,18 @@ const (
 	upgradeStateKCPUpgraded           = "KCPUpgraded"
 	upgradeStateMDPatchApplied        = "MDPatchApplied"
 	upgradeStateSuccess               = "Success"
+	vipLeaseDurationKey               = "VIP_LEASEDURATION"
+	vipRenewDeadlineKey               = "VIP_RENEWDEADLINE"
+	vipRetryPeriodKey                 = "VIP_RETRYPERIOD"
+	vipLeaseDuration                  = "vip_leaseduration"
+	vipRenewDeadline                  = "vip_renewdeadline"
+	vipRetryPeriod                    = "vip_retryperiod"
+	defaultLeaseDuration              = "15"
+	defaultRenewDeadline              = "10"
+	defaultRetryPeriod                = "2"
+	defaultNewLeaseDuration           = "30"
+	defaultNewRenewDeadline           = "20"
+	defaultNewRetryPeriod             = "4"
```

Comment:
can we define these constants as part of the different file or as a new `const` with a comment within this file, as other constants that are mainly related to `upgradeState`?

Reply:
Sure thing. Moved these and the `kubevip` related functions to another file